imagej/src/main/java/science/aist/imaging/service/imagej/imageprocessing/wrapper/ImageStackWrapper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 the original author or authors.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package science.aist.imaging.service.imagej.imageprocessing.wrapper;
+
+import ij.ImageStack;
+import science.aist.imaging.api.domain.wrapper.AbstractImageWrapper;
+import science.aist.imaging.api.domain.wrapper.ChannelType;
+
+/**
+ * <p>Created by Christoph Praschl on 11.02.2021</p>
+ * <p>Implementation of a {@link science.aist.imaging.api.domain.wrapper.ImageWrapper} for ImageJ's {@link ImageStack}</p>
+ *
+ * @author Christoph Praschl christoph.praschl@fh-hagenberg.at
```

Comment:
- Remove created by
- remove email address
- add since tag.

Reply:
Fixed in [739c19f](https://github.com/FHOOEAIST/imaging/pull/14/commits/739c19fa2d18dad40ebac4676254c36ff07279f0)